blueprints/server/index.js

```diff
@@ -32,8 +33,18 @@ module.exports = {
   },
 
   hasJSHint() {
-    if (this.project) {
-      return 'ember-cli-jshint' in this.project.dependencies();
-    }
+    let hasJSHint = Boolean(this.project) && 'ember-cli-jshint' in this.project.dependencies();
+
+    deprecate('Support for `ember-cli-jshint` is deprecated. We recommend using `eslint` instead.', !hasJSHint, {
+      for: 'ember-cli',
+      id: 'ember-cli.ember-cli-jshint-support',
+      since: {
+        available: '4.5.0',
```

Comment:
I think this would be 4.6.0 at this point, unless you targeted beta.

Reply:
I might be wrong since we haven't done 4.4 yet.

Reply:
Updated to `4.6.0` since `4.4.0` is now released.